code-review-trainer-app/yarn.lock

```diff
@@ -4,32 +4,32 @@
 
 "@ampproject/remapping@^2.2.0":
   version "2.3.0"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.3.0.tgz#ed441b6fa600072520ce18b43d2c8cc8caecc7f4"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.3.0.tgz"
```

Comment:
I wouldn't expect yarn.lock updates when this commit is only adding css styles... what happened here?